python/llm/example/GPU/Speculative-Decoding/Eagle/evaluation/speed.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2016 The BigDL Authors.
+# 
```

Comment:
Did you modify the original speed.py from eagle github repo? If not, we don't need bigdl copyright here

Reply:
Speed.py has been modified, so it is different from the speed.py in eagle github repo.